csharp/src/Ice/EndpointFactory.cs

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace ZeroC.Ice
+{
+    internal class EndpointFactory : IEndpointFactory
+    {
+        private Communicator Communicator { get; }
```

Comment:
Should this be a private _communicator readonly data member?

Reply:
fixed